### DISCUSSION TITLE
how i can delete blender, krita...

### DISCUSSION THREAD

**Discussion by nuexq** [prasanthrangan/hyprdots#1094] [Q&A] [answered]
I doesn't need some apps and i want to uninstall them but i doesn't know how, i check paru and pacman and it's not installed with theme

**Comment by kRHYME7** [comment, marked as answer]
> blender, krita 

Who installed this then? 

try to see if it's on flatpak list. 

``` flatpak list ```

**Reply by nuexq** [reply]
it's installed by default when you install hyprdots , and thanks for information it's work
